src/main/MQ2Main.vcxproj.filters

```diff
@@ -711,4 +711,4 @@
       <Filter>Source Files</Filter>
     </NASM>
   </ItemGroup>
-</Project>
+</Project>
```

Comment:
unnecessary change

Reply:
I cannot see this change locally so I am not sure how to fix it.